ci-operator/templates/openshift/openshift-azure/build-node-image.yaml

```diff
@@ -18,7 +18,7 @@ parameters:
   required: true
 - name: OPENSHIFT_RELEASE
   value: "3.11"
-  required: true
+- name: AZURE_OREG_URL
```

Comment:
seems like this should be required

Reply:
if not passed playbooks should just ignore it.

Reply:
ok